runtime-rocm/rocm-rocblas/autobuild/defines

```diff
@@ -0,0 +1,36 @@
+PKGNAME=rocm-rocblas
+PKGDES="rocBLAS is the AMD library for BLAS in ROCm. Implemented using the HIP language and optimized for AMD GPUs"
+PKGSEC=devel
+PKGDEP="rocm-llvm rocm-clr rocm-cmake rocm-hipblas-common rocm-roctracer rocm-hipblaslt"
+BUILDDEP="cmake cargo rustc llvm maturin msgpack-c++"
+
+# FIXME gfortran unsupported ld.lld, BUILD_CLIENTS_* requirement fortran(gfortran)
+NOLTO=1
+
+USECLANG=1
+AB_FLAGS_CFP=0
+
+ABTYPE=cmakeninja
+CMAKE_AFTER=(
+    '-DCMAKE_INSTALL_PREFIX=/usr/lib/rocm'
+    '-DCMAKE_C_COMPILER=/usr/lib/rocm/lib/llvm/bin/clang'
+    '-DCMAKE_CXX_COMPILER=/usr/lib/rocm/lib/llvm/bin/clang++'
+    '-DCMAKE_SKIP_INSTALL_RPATH=OFF'
+    '-DCMAKE_PREFIX_PATH=/usr/lib/rocm'
+    '-DROCM_PATH=/usr/lib/rocm'
+    '-DBUILD_CLIENTS_SAMPLES=ON'
+    '-DBUILD_CLIENTS_TESTS=OFF'
+    '-DBUILD_CLIENTS_BENCHMARKS=OFF'
+    '-DBUILD_WITH_HIPBLASLT=ON'
+    '-DRUN_HEADER_TESTING=OFF'
+    '-DGPU_TARGETS=gfx1030;gfx1100;gfx1101;gfx1102;gfx1200;gfx1201'
+    '-DCMAKE_FIND_ROOT_PATH=/usr/lib/rocm/;/usr/lib/rocm/lib/llvm/'
+    '-DCMAKE_INSTALL_RPATH=/usr/lib/rocm/lib/llvm/lib;/usr/lib/rocm/lib'
+)
+
+BUILD_FINAL(){
+  ninja rocblas-common
```

Comment:
Inconsistent indentation, please keep them the same throughout - you seem to prefer 4 spaces.